packages/@uppy/dashboard/src/components/FileCard.js

```diff
@@ -49,10 +42,8 @@ class FileCard extends Component {
           placeholder={field.placeholder}
           onkeyup={this.tempStoreMetaOrSubmit}
           onkeydown={this.tempStoreMetaOrSubmit}
-          onkeypress={this.tempStoreMetaOrSubmit}
-          ref={(el) => {
-            if (i === 0) this.firstInput = el
-          }} /></fieldset>
+          onkeypress={this.tempStoreMetaOrSubmit} />
+      </fieldset>
```

Comment:
Why is there no `data-uppy-super-focusable` here, will it focus on the first input?